pkg/cmd/roachtest/cluster.go

```diff
@@ -2104,6 +2104,26 @@ func (c *clusterImpl) configureClusterSettingOptions(
 	}
 }
 
+// Grow adds nodes to the cluster.
+func (c *clusterImpl) Grow(ctx context.Context, l *logger.Logger, nodeCount int) error {
+	err := roachprod.Grow(ctx, l, c.name, c.IsSecure(), nodeCount)
+	if err != nil {
+		return err
+	}
+	c.spec.NodeCount += nodeCount
+	return nil
+}
+
+// Shrink removes nodes from the cluster.
+func (c *clusterImpl) Shrink(ctx context.Context, l *logger.Logger, nodeCount int) error {
+	err := roachprod.Shrink(ctx, l, c.name, nodeCount)
```

Comment:
True, in my mind shrink should only be useful when everything is "okay" and we want to free up resources. In the context of a `roachtest` it will probably never make sense to call shrink, but rather just stop cockroach etc, and it will likely be at the end of a test. IMHO I'm not sure if there's much value for these in the context of roachtest and we are not entering a space where operations/roachtests don't quite overlap.